examples/simple/README.md

```diff
@@ -4,8 +4,12 @@
 
 ### Install
 
+Start fetchr root:
+
 ```js
 npm install
+cd examples/simple
+npm install
```

Comment:
@redonkulus If you remove the node_modules folder in the root and then follow the original instructions you'll see that they fail. i.e. the examples depend on `npm i` having been run in the root *and* in the examples/simple folder before it will work.